src/main/java/com/tarento/assetManagement/dao/dto/Audit.java

```diff
@@ -0,0 +1,18 @@
+package com.tarento.assetManagement.dao.dto;
+
+import java.util.Date;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Audit {
+
+	private Integer id;
+	private Date auditDate;
```

Comment:
add one more attribute as assetId  as int .

Reply:
id is mapped to assetId 
Should i add new attribute or rename the same?

Reply:
I feel it would be good if u can create an id as primary-key which is auto increment and another attribute as assetId which will be foreign-key from asset table.